tests/integration/snowflake_project/definitions/sample_data_2.sqlx

```diff
@@ -1,4 +1,4 @@
-config { type: "view" }
+config { type: "view", database: "TADA2" }
```

Comment:
Does the snowflake integration test actually check if we created this successfully?